src/__tests__/gameHandlerClass.spec.ts

```diff
@@ -0,0 +1,129 @@
+import { GameHandler } from "../gameHandlerClass";
+import { Player } from "../playerClass";
+import { PokemonFactory } from "../pokemonFactory";
+import { PokemonType } from "../pokemonClass";
+import * as data from "../../assets/poke_data.json";
+
+describe("Test Game Handler class", () => {
+  const pokeData = data.pokemons;
+
+  it("GameHandler.isGameFinished should return false if every player has at least one pokemon", () => {
+    // Given
+    const factory_0 = new PokemonFactory(pokeData);
```

Comment:
what's with this naming convention?

each test is separated from others, so there is no need for this suffix `_0`, `_1` etc.